akka-testkit/src/main/mima-filters/2.6.3.backwards.excludes/28604-fix-ExplicitlyTriggeredScheduler-cancellation.excludes

```diff
@@ -0,0 +1,8 @@
+# #28604 changes in the akka.testkit.ExplicitlyTriggeredScheduler internal implementation.
+# Removal of the `time` field from the internal Item case class in ExplicitlyTriggeredScheduler.
+ProblemFilters.exclude[IncompatibleMethTypeProblem]("akka.testkit.ExplicitlyTriggeredScheduler#Item.apply")
```

Comment:
:+1:, FYI you can use wildcards in mima, so for example:
```suggestion
ProblemFilters.exclude[IncompatibleMethTypeProblem]("akka.testkit.ExplicitlyTriggeredScheduler#Item.*")
```
but this is fine

Reply:
Thanks for the hint, I changed that 👍